pkgs/applications/editors/kakoune/wrapper.nix

```diff
@@ -1,44 +1,32 @@
-{ stdenv, bash }:
-with stdenv.lib;
-
-kakoune:
+{ symlinkJoin, makeWrapper, kakoune, plugins ? [], configure ? {} }:
 
 let
-  getPlugins = { plugins ? [] }: plugins;
+  # prefer "plugins" over "configure.plugins", but both work
+  requestedPlugins =
+    if builtins.length plugins > 0
+      then plugins
+      else configure.plugins or [];
```

Comment:
Why not just concatenate the two? This seems the lest surprising and simplest.

Reply:
Makes sense, changed this